### PR TITLE
Config log

### PR DESCRIPTION
**Pull request by mdnazmulkarim**
Make DebugMap configurable where used in ruler

### CODE REVIEW

**Comment by JPercival** [comment]
Let's call the logging property `hapi.fhir.cql_logging_enabled' to better align with the standards in hapi fhir:

https://github.com/hapifhir/hapi-fhir-jpaserver-starter/blob/master/src/main/resources/application.yaml#L70

Note that the hapi fhir server uses a newer process for setting config that we've not yet adopted, so the way the config is done will need to change to migrate this back up to hapi. I'm not worried about that at the moment because there's quite a bit of work to do to get the cqf-ruler aligned with hapi_fhir_jpaserver_starter. My plan is to cut a release of the cqf-ruler before doing all that refactoring.

**Comment by JPercival** [comment]
Also, OHSU is working on some logback config that'll also be necessary for the logging to actually show up.

**Review by sliver007 — APPROVED** [review]
Looks good to me.

File: dstu3/src/main/resources/logback.xml [hunk 1]
```diff
@@ -37,4 +37,9 @@
         <!-- <appender-ref ref="htmlAppender" /> -->
     </root>
 
+    <logger name = "org.opencds.cqf.dstu3" level="DEBUG" additivity="false">
```
Comment (JPercival) [inline dstu3/src/main/resources/logback.xml:40]: We're targeting CQL engine logging specifically, so this name should probably be:
`org.opencds.cqf.cql.engine.debug`

File: r4/src/main/resources/logback.xml [hunk 2]
```diff
@@ -37,4 +37,9 @@
         <!-- <appender-ref ref="htmlAppender" /> -->
     </root>
 
+    <logger name = "org.opencds.cqf.r4" level="DEBUG" additivity="false">
```
Comment (JPercival) [inline r4/src/main/resources/logback.xml:40]: Same as above: `org.opencds.cqf.cql.engine.debug`

**Pull request merged**